conbench/entities/summary.py

```diff
@@ -148,6 +151,18 @@ def create(data):
             bulk.append(Time(result=x, summary_id=summary.id, iteration=i + 1))
         Time.bulk_save_objects(bulk)
 
+        distribution = get_distribution(repository, sha, 1000).first()
```

Comment:
Am I reading this + following the logic in the function calls that this will get a distribution based on the last 1000 commits (as specified in our index)? 

What's the `.first()` doing there? I'm sure it's right — but curious what's going on with it?

Reply:
I use `first` here because I know it will also be a list of size 1, and I don't want to have to always go `distribution 
 = distribution[0]`, first does that for me.

Reply:
PS. Yes, 1000 commits from the index.

Reply:
Cool, more is almost certainly better, so we can leave it as is, but I was testing this with 100 commits and it was doing a decent job of capturing the variation. Happy to leave it at 1000, but if we run into any bottlenecks, feel free to drop that limit lower